### PR TITLE
LPS-101085 Apply view count service to KBArticle

### PR DESCRIPTION
**Pull request by stsquared99**

### CODE REVIEW

**Comment by stsquared99** [comment]
Tests passed here: https://github.com/stsquared99/liferay-portal/pull/89

**Comment by Preston-Crary** [comment]
Please see https://github.com/shuyangzhou/liferay-portal/pull/8586

File: modules/apps/knowledge-base/knowledge-base-service/src/main/java/com/liferay/knowledge/base/service/impl/KBArticleLocalServiceImpl.java [hunk 1]
```diff
@@ -1318,32 +1358,6 @@ public KBArticle updateStatus(
 		return kbArticle;
 	}
 
-	@Override
-	public void updateViewCount(
```
Comment (Preston-Crary) [inline modules/apps/knowledge-base/knowledge-base-service/src/main/java/com/liferay/knowledge/base/service/impl/KBArticleLocalServiceImpl.java]: For changes like this separate renaming / sorting of the method so there's a clear diff on the method body.  Makes it easier for reviewers to see what really changed and separates logic changes for naming changes.

Thanks!

Comment (Preston-Crary) [reply modules/apps/knowledge-base/knowledge-base-service/src/main/java/com/liferay/knowledge/base/service/impl/KBArticleLocalServiceImpl.java]: Usually I will rename with the logic change and sort afterwards.

**Pull request closed without merging**